ui/component/commentReactions/view.jsx

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as ICONS from 'constants/icons';
+import * as REACTION_TYPES from 'constants/reactions';
+import React from 'react';
+import classnames from 'classnames';
+import Button from 'component/button';
+import usePersistedState from 'effects/use-persisted-state';
+
+type Props = {
+  myReacts: Array<string>,
+  othersReacts: any,
+  react: (string, string) => void,
+  commentId: string,
+};
+
+export default function CommentReactions(props: Props) {
+  const { myReacts, othersReacts, commentId, react } = props;
+  const [activeChannel] = usePersistedState('comment-channel');
```

Comment:
This is really cool. I never thought of using `usePersistedState` like this!

Reply:
pretty sure I copied it from somewhere else